README.md

```diff
@@ -1,4 +1,11 @@
 # README
+Authors:
+
+    Matheus Lopez https://github.com/matheus-fls
+    Anthony Tapia https://github.com/AnthonyTC89
+
+Project: Building With Active Record
```

Comment:
One should normally be a little more descriptive on this file, but since it's a small project I'm going to let this one pass. Feel free to update this with what you learned and/or the objective of this project, including how to get set up with `bundle install`, running `rails db:setup`, creating a user, post, comment and checking for associations in the console like `User.first.posts`, `User.second.comments`, `Post.first.comments`, etc.